samples/snippets/src/test/java/pubsub/SchemaIT.java

```diff
@@ -73,12 +73,8 @@ private static void requireEnvVar(String varName) {
         System.getenv(varName));
   }
 
-  @Rule public Timeout globalTimeout = Timeout.seconds(300); // 5 minute timeout
-
-  @BeforeClass
```

Comment:
is this no longer needed?

Reply:
Oops, I think I deleted these by accident.